packages/flutter_tools/test/commands.shard/hermetic/generate_localizations_test.dart

```diff
@@ -466,7 +466,7 @@ format: true
     pubspecFile.writeAsStringSync('''
   name: test
   environment:
-    sdk: '>=3.0.0-0 <4.0.0'
+    sdk: '>=3.2.0-0 <4.0.0'
```

Comment:
did you just grep to find all these magic strings? outside of this PR, maybe we should just have a global const string to make bumping this easier.

Reply:
That's what I did, yeah. That sounds like a good idea!